charts/consul/templates/terminating-gateways-deployment.yaml

```diff
@@ -97,6 +97,9 @@ spec:
         - name: consul-service
           emptyDir:
             medium: "Memory"
+        - name: consul-data
```

Comment:
We have used `consul-data` in some existing helm charts like mesh gateway and others and in other work like client token others, we have used `consul-login`. I will look at trying to align those outside of this PR.